### PR TITLE
Adding K3d installation instructions.

### PR DESCRIPTION
**Pull request by subhamgcon**
Solves issue #1121. 

Signed-off by Subham Ghosh: subhamgcon@gmail.com

### CODE REVIEW

File: content/docs/0.13.x/operate/install/index.md [hunk 1]
```diff
@@ -185,6 +185,35 @@ Please refer to the [official homepage of K3s](https://k3s.io) for detailed inst
...
+
+ 
+1. Download, install [K3d](https://k3d.io/v5.3.0/) (tested with [v5.3.0](../k8s_support)) and run K3d using the following command:
+
+   ```console
+   curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=v5.3.0 bash
+   ```
+   This installs version `v5.3.0` (please refer to the [K3d GitHub releases page](https://github.com/k3d-io/k3d/) for newer releases).
+
+1. Create a cluster called My keptn which will have port fowarding also will disable the traffic which is a ingress gateaway.
```
Comment (StackScribe) [inline content/docs/0.13.x/operate/install/index.md:204]:
```suggestion
1. Create a cluster called My keptn which has port fowarding and disables the traffic, which is a ingress gateaway.
```
Present tense is usually better than future.

File: content/docs/0.13.x/operate/k8s_support/index.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
 * **EKS** ... Amazon Elastic Kubernetes Service
 * **GKE** ... Google Kubernetes Engine
 * **K3s** ... A certified Kubernetes distribution built for IoT & Edge computing: [k3s.io](https://k3s.io/)
+* **K3d** ... k3d is a lightweight wrapper to run k3s (Rancher Lab’s minimal Kubernetes distribution) in docker: [k3d.io](https://k3d.io/v5.3.0/)
```
Comment (StackScribe) [inline content/docs/0.13.x/operate/k8s_support/index.md:45]:
```suggestion
* **K3d** ... A lightweight wrapper that runs k3s (Rancher Lab’s minimal Kubernetes distribution) in Docker: [k3d.io](https://k3d.io/v5.3.0/)
```
Keep this consistent with the other items in this list, which do not restate the item's name.

Comment (subhamgcon) [reply content/docs/0.13.x/operate/k8s_support/index.md]: Okay!

File: content/docs/0.13.x/operate/install/index.md (hunk 1 above)
Comment (subhamgcon) [reply content/docs/0.13.x/operate/install/index.md]: Understood!

File: content/docs/0.13.x/operate/install/index.md [hunk 3]
```diff
@@ -185,6 +185,35 @@ Please refer to the [official homepage of K3s](https://k3s.io) for detailed inst
 </p>
 </details>
 
+<details><summary>K3d</summary>
+<p>
+
+Please refer to the [official homepage of K3d](https://k3d.io/v5.3.0/) for detailed installation instructions. Here, a short guide on how to run Keptn on K3d is provided for a Linux environment. **Note:** [Docker](https://docs.docker.com/get-docker/) is required to be able to use k3d at all
+Note: k3d v5.x.x requires at least Docker v20.10.5 (runc >= v1.0.0-rc93) to work properly.
+[kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the Kubernetes cluster
```
Comment (StackScribe) [inline content/docs/0.13.x/operate/install/index.md:193]:
```suggestion
Please refer to the [official homepage of K3d](https://k3d.io/v5.3.0/) for detailed installation instructions. Here, a short guide on how to run Keptn on K3d is provided for a Linux environment.

**Note:** [Docker](https://docs.docker.com/get-docker/) is required to use k3d.
k3d v5.x.x requires at least Docker v20.10.5 (runc >= v1.0.0-rc93) to work properly.

You must install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) before installing K3d. This is used to interact with the Kubernetes cluster.
```

**Review by StackScribe — COMMENTED** [review]
A few editorial suggestions but, in general, lgtm.

**Comment by subhamgcon** [comment]
> A few editorial suggestions but, in general, lgtm.

Thank you for reviewing! I have committed your suggested changes. Is it ready to be merged then?

File: content/docs/0.13.x/operate/install/index.md (hunk 3 above)
Comment (subhamgcon) [reply content/docs/0.13.x/operate/install/index.md]: Done! Thanks for the edits.

**Review by StackScribe — APPROVED** [review]

**Comment by StackScribe** [comment]
It needs review by a technical person but I approved it.

**Comment by subhamgcon** [comment]
@oleg-nenashev. Can you review it? Thank you!

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Thanks! Needs a pull request for 0.14.x, Maybe we could also restructure this part and start adding top-level installation guide with version-unspecific guidelines

**Review by subhamgcon — COMMENTED** [review]
Can you please explain the requested changes? Also, should I start working on a PR for 0.14.x?

**Comment by oleg-nenashev** [comment]
>  should I start working on a PR for 0.14.x?

You can update it in the same pull request. I think we can try reworking it later as experiment together with @StackScribe

**Review by oleg-nenashev — APPROVED** [review]
Thanks! I think it is a good improvement

**Pull request merged**